lib/src/cupertino/cupertino_toolbar.dart

```diff
@@ -565,7 +651,7 @@ class _RenderIosPagedMenu extends RenderBox
     double currentPageWidth = 0.0;
     double buttonsWidth = previousButtonSize.width + nextButtonSize.width;
 
-    for (int i = 1; i < children.length; i++) {
+    for (int i = 1; i < children.length - 1; i++) {
```

Comment:
Please add a comment about why we're starting at `1` and we're stopping one before the end.